### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cforbe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 
 # Azure Data Preparation Public Preview
 
+The Azure Machine Learning Data Prep SDK is used to load, transform, and write data for machine learning workflows. You can interact with the SDK in any Python environment, including Jupyter Notebooks or your favorite Python IDE. The Azure Machine Learning Data Prep SDK includes the following functionality to help prepare your data for modeling:
```
Comment (yikei) [inline README.md]: "the following functionality" - probably should be "the following set of functionalities" for plural

File: README.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+
+- Intelligent transforms. Use the SDK to derive or split a column by example, impute missing values, fuzzy group, auto join, and perform other automated tasks.
```
Comment (yikei) [inline README.md]: Comment for all the dot points:
Most of these dot points are structured like "title of dot point" + "elaboration of dot point". Can you either separate them into sub-points, or rephrase the sentences to improve the flow?

Comment (cforbe) [reply README.md]: @yikei I'm going to just make them ':' e.g. title: explanation

File: README.md [hunk 3]
```diff
@@ -1,6 +1,15 @@
 
 # Azure Data Preparation Public Preview
 
+The Azure Machine Learning Data Prep SDK is used to load, transform, and write data for machine learning workflows. You can interact with the SDK in any Python environment, including Jupyter Notebooks or your favorite Python IDE. The Azure Machine Learning Data Prep SDK includes the following set of functionalities to help prepare your data for modeling:
+
+- Intelligent transforms: Use the SDK to derive or split a column by example, impute missing values, fuzzy group, auto join, and perform other automated tasks.
+- Smart reading functionality: The SDK can automatically detect any of the supported file types. You don’t need to use special file readers for CSV, text, Excel, etc., or to specify delimiter, header, or encoding parameters.
+Varying schema processing. The SDK engine can read different columns per row instance, also sometimes referred to as ragged-right format.
```
Comment (yikei) [inline README.md]:
```suggestion
- Varying schema processing: The SDK engine can read different columns per row instance, also sometimes referred to as ragged-right format.
```

**Comment by cforbe** [comment]
@yikei

File: README.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
 
 # Azure Data Preparation Public Preview
 
+The Azure Machine Learning Data Prep SDK is used to load, transform, and write data for machine learning workflows. You can interact with the SDK in any Python environment, including Jupyter Notebooks or your favorite Python IDE. The Azure Machine Learning Data Prep SDK includes the following set of functionalities to help prepare your data for modeling:
+
+- Intelligent transforms: Use the SDK to derive or split a column by example, impute missing values, fuzzy group, auto join, and perform other automated tasks.
+- Smart reading functionality: The SDK can automatically detect any of the supported file types. You don’t need to use special file readers for CSV, text, Excel, etc., or to specify delimiter, header, or encoding parameters.
+Varying schema processing: The SDK engine can read different columns per row instance, also sometimes referred to as ragged-right format.
```
Comment (yikei) [inline README.md]: My previous comment had 2 changes.
1) "Varying schema processing" needs to be its own dot point (add a minus sign at the beginning).
2) Change the full stop to a colon.

Your most recent change addressed #2, not #1.

**Pull request merged**